evap/staff/templates/staff_semester_view.html

```diff
@@ -377,6 +377,12 @@ <h3 class="m-0 me-1">{{ semester.name }}</h3>
                     </div>
                 </div>
 
+                {% for evaluation in evaluations %}
+                    <form id="exam_creation_form_{{ evaluation.id }}" reload-on-success method="post" action="{% url 'staff:create_exam_evaluation' %}">
```

Comment:
Given that except for the `id`, all these forms are equal, we really only need one

Reply:
We concluded in person that this is needed because the date-selection inputs in all the modals should be tied to different forms. Please document this decision in a comment here though :)

Reply:
Unresolved, don't see a comment

> Please document this decision in a comment here though :)

Reply:
As Niklas said the code is needed and his initial comment is not relevant anymore

Reply:
Please add a short comment in the code, something like
```django
{# separate forms for each modal since we want separate date-selection inputs because REASON #}
```